packages/core/src/checkbox/checkbox-group.ts

```diff
@@ -57,6 +66,7 @@ export class CheckboxGroup extends HintMixin(LitElement) {
 
   /** Custom validators. */
   @property({ attribute: false }) validators?: Validator[];
+  @property() name?: string;
```

Comment:
Please add a comment for each property. That way, those comments are automatically used as API documentation.